equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm14Controller.java

```diff
@@ -37,6 +38,17 @@ public ResponseEntity<Crm14FormDTO> getApplicationCrm14(Long usn, String profile
         Crm14Model crm14FormData = crmFileService.getCrmFormData(crmFormDetailsCriteriaDTO);
         List<Crm14AttachmentModel> attachments = crm14AttachmentService.getCrm14Attachments(usn);
         crm14FormData.getFormDetails().setProcessedAttachments(attachments);
+        addProcessedAttachmentsToEvidence(crm14FormData,attachments);
         return ResponseEntity.ok(mapper.getDTOFromModel(crm14FormData));
     }
+
+    public void addProcessedAttachmentsToEvidence(Crm14Model crm14FormData, List<Crm14AttachmentModel> processedAttachments) {
```

Comment:
I think this function can be moved to the Crm14Model class. Or maybe just to the Crm14AttachementService class. 
Maybe both.. so the service calls the model one? 
But it any option would be better than having this on the controller because it is more business logic

Reply:
This can be a follow up ticket